.codeclimate.yml

```diff
@@ -3,7 +3,7 @@ engines:
     enabled: true
   eslint:
     enabled: true
-    channel: "eslint-5"
+    channel: "eslint-4"
```

Comment:
@koooge can this cause codeclimate redundant warnings?

Reply:
Yes, it causes lots of `html-webpack-plugin was not found` error. It seems a codeclimate-eslint's issue.

Reply:
I mean the other way around - will the downgrade create discrepancies between the codebase linting (v5) and codeclimate's results (v4)?

Reply:
Oh, it's absolutely yes. But I think it's not a big problem since codeclimate can disable specific rules.
https://github.com/getredash/redash/blob/master/.codeclimate.yml#L9-L11
Actually, this PR upgrades codeclimate's eslint3 to 4, and codebase's eslint4 to 5. So, their vesions are already different.